mslib/msui/viewwindows.py

```diff
@@ -140,6 +142,8 @@ def createDockWidget(self, index, title, widget):
         # setWidget transfers the widget's ownership to Qt -- no setParent()
         # call is necessary:
         self.docks[index].setWidget(widget)
+        # store data of widget
+        self.widgets[index] = widget
```

Comment:
I've not seen that this can be accessed anywhere else. This enables now also other tests for the docking widgets.

Reply:
Wouldn't `self.docks[n].widget() == self.widgets[n]`? (https://doc.qt.io/qt-6/qdockwidget.html#widget)

I.e. can't you just get the widget out of the dock?

Reply:
Good idea, also in QT-5 available.  That simplifies it a lot.